apps/gdalalg_vector_concat.cpp

```diff
@@ -238,13 +282,42 @@ bool GDALVectorConcatAlgorithm::RunStep(GDALPipelineStepRunContext &)
     else if (m_sourceLayerFieldName.empty())
         m_sourceLayerFieldName = "source_ds_lyr";
 
+    const int nMaxSimultaneouslyOpened =
+        std::max(atoi(CPLGetConfigOption(
+                     "GDAL_VECTOR_CONCAT_MAX_OPENED_DATASETS", "100")),
+                 1);
+
     // First pass on input layers
     std::map<std::string, std::vector<LayerDesc>> allLayerNames;
     int iDS = 0;
+    int countNonOpenedDS = 0;
```

Comment:
A verb at the beginning makes me think of a function but maybe it's just me.
```suggestion
    int nonOpenedDSCount = 0;
```

Reply:
renamed as suggested